client/src/pages/docs/SharkioDocsSetupPage.tsx

```diff
@@ -36,7 +37,8 @@ export const SharkioDocsSetupPage = () => {
         <img
           src="/apiKeys.png"
           alt=""
-          className="w-full rounded-lg h-96 border-2 border-border-color mt-4"
+          className="rounded-lg border-2 border-border-color mt-4"
+          style="width:90%"
```

Comment:
Could you use w-[90%] instead?

Reply:
Sure. Added `w-[90%]` instead of that. Please let me know if you have any other questions / want to change something else!

Reply:
You are welcome to join our community on discord - https://discord.gg/ca78sanYBK.
If you want to continue the contribution, let's discuss about it - my discord username is yairharas.